SECURITY.md

```diff
@@ -0,0 +1,9 @@
+# Security Policy
+
+## Supported Versions
+
+Uyuni follows a rolling release model, so it is always the next upcoming version that will contain bugfixes for the current one as well as any new features. As there will be no maintenance of specific versions, users will always need to update to the most recent version in order to receive the latest security updates.
```

Comment:
In very specific cases, we release patches, for very critical bugfixes in terms of impact, or for some CVEs, again according to the impact.

That's a case by case decision.

Reply:
Would it make sense to tie such a decision to concrete [CVSS](https://en.wikipedia.org/wiki/Common_Vulnerability_Scoring_System) rating? Alternatively we could re-use SUSE's simplified rating https://www.suse.com/support/security/rating/ and say e.g. important & critical flaws are fixed by patches, moderate & low flaws are fixed by rolling releases.

The classification still is a case by case decision, just guided by a framework that helps us with the classification and sets expectations straight for our users.

Reply:
Thanks Alex, I like the idea, especially given the simplified ratings. After talking to Julio though, it seems he would rather decide this case by case for now instead of giving a promise, so I changed the text correspondingly.

Reply:
What we discussed was a bit different. Let's talk again on Monday, this time including @agraul.

Reply:
Updated the text after our latest conversation and corresponding with @thesp0nge.